### DISCUSSION TITLE
第三者レビュー ログ-開発ガイド

### DISCUSSION THREAD

**Discussion by creature-water-valley** [ws-4020/mobile-app-crib-notes#570] [General]
#### ログの利用

- 開発時のデバッグ用途の利用
  - 「デバッグ用途の利用」ではなく「デバッグ用途に利用」がよさそうに思いました
- ソースコードとのマッピング
  - Androidの場合はWindows環境もあるとは思うので、`cat`コマンドがあるのか気になりました

**Comment by creature-water-valley** [comment]
### 対応内容

- s/デバッグ用途の利用/デバッグ用途に利用/
- 注意書き（※）でtypeコマンドで代用を書く

**Comment by kozake** [comment]
### 対応PR

https://github.com/ws-4020/mobile-app-crib-notes/pull/565

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
